03-make/make_soup/makefile

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/make -f
+
+#variables for hirerarchy
+KITCHEN = kitchen
+KNIFE = ${KITCHEN}/knife
+TABLE = ${KITCHEN}/table
+POT = ${TABLE}/pot
+SPOON = ${TABLE}/spoon
+DINNING_ROOM = dinning-room
+FOOD_MARKET = food-market
+
+#files with goods
+POTATO = potato
+WATER = water
+CARROT = carrot
+
+#commands
+CREATE_DIR = -mkdir
+REMOVE = rm -rf
+LIST_DIR = ls -la
+CAT = -cat
+
+MOVE_SCRIPT = ./move_script.sh
+
+#default count
+DEFAULT_ITEMS_CNT=10
+
+all: ${POT}/${POTATO} ${POT}/${WATER} ${POT}/${CARROT}
+	echo Soup is done
+	${REMOVE} $^
+
+${POT}/%: ${TABLE}/% ${MOVE_SCRIPT}
+	#move from table to pot
+	${MOVE_SCRIPT} $< $@
+
+${TABLE}/%: ${FOOD_MARKET}/% ${MOVE_SCRIPT}
+	#move from market to table
+	${MOVE_SCRIPT} $< $@
+
+${MOVE_SCRIPT}:
+	echo '#!/bin/bash' > ${MOVE_SCRIPT}
```

Comment:
It is better use compound command like () > $(MOVE_SCRIPT}
or even better
cat > $script <<EOF
you multiline
text
EOF

Reply:
It seems it doesn't work. I tried to copy-paste your example to my make file and there is no such test in move_script.sh.